docs/user-guide/src/quick-start.md

```diff
@@ -0,0 +1,916 @@
+# Quick-start for Metal3
+
+This guide has been tested on Ubuntu server 22.04. It should be seen as an
+example rather than the absolute truth about how to deploy and use Metal3. We
+will cover two environments and two scenarios. The environments are
+
+1. a baremetal lab with actual physical servers and baseboard management
+   controllers (BMCs), and
+2. a virtualized baremetal lab with virtual machines and sushy-tools acting as
+   BMC.
+
+In both of these, we will show how to use Baremetal operator and Ironic to
+manage the servers through a Kubernetes API, as well as how to turn the servers
+into Kubernetes clusters managed through Cluster API. These are the two
+scenarios.
+
+In a nut-shell, this is what we will do:
+
+1. Setup a management cluster
+2. Setup a DHCP server
+3. Setup a disk image server
+4. Deploy Ironic
+5. Deploy Baremetal operator
+6. Create BareMetalHosts to represent the servers
+7. (Scenario 1) Provision the BareMetalHosts
+8. (Scenario 2) Deploy Cluster API and turn the BareMetalHosts into a Kubernetes
+   cluster
+
+## Prerequisites
+
+You will need the following software installed.
+
+- docker (or podman)
```

Comment:
Docker as a softwarere is a name, hence it would need to be capitalized. docker as a CLI command would not. So either fix the capitalization of all the tools, or alternatively L31 could read tools instead of software?